app/services/search/postgres/feed.rb

```diff
@@ -0,0 +1,103 @@
+module Search
+  module Postgres
+    class Feed
+      FIELDS_TO_EXCLUDE_FROM_SERIALIZATION = %i[
+        approved body_text experience_level_rating
+        experience_level_rating_distribution featured featured_number
+        hotness_score language published reactions_count score
+      ].freeze
+
+      def self.search_documents(
+        term: nil, approved: nil, class_name: nil, id: nil, sort_by: nil, sort_direction: nil, published_at: nil,
+        page: 1, per_page: 60
+      )
+        page = (page || 1).to_i + 1 # NOTE: the frontend starts at 0, need to fix this
+        per_page = [(per_page || 60).to_i, 100].min
+
+        relation = if term.present?
+                     relation = Search::Multisearch.call(term)
+                       .with_pg_search_highlight
+                       .with_pg_search_rank
+                       .includes(
+                         searchable: [
+                           {  podcast_episode: {  podcast: :creator } },
+                           { articles: :organization },
+                           :tag_taggings, :tags, :comments
+                         ],
+                       )
+
+                     relation = select_approved_articles(relation, approved, class_name)
+                     relation = relation.where(searchable_type: class_name) if class_name.present?
+                     relation.where(searchable_id: id) if id.present?
+                   else
+                     relation = PgSearch::Document
+                     relation = relation.where(searchable_type: class_name) if class_name.present?
+                     select_articles_by_publication_date(relation, published_at, class_name)
+                   end
+
+        relation = sort_documents(relation, sort_by, sort_direction)
+
+        relation.page(page).per(per_page).map do |doc|
+          searchable = doc.searchable
+          serializer = ::Search.const_get("#{searchable.class}Serializer")
+          attributes = serializer.new(searchable).serializable_hash.dig(:data, :attributes)
+          prepare_doc(doc, attributes).as_json
+        end
+      end
+
+      # borrowed from Search::FeedContent.prepare_doc
+      def self.prepare_doc(doc, attributes)
+        attributes[:_score] = doc.respond_to?(:pg_search_rank) ? doc.pg_search_rank : 0
```

Comment:
this is because search ranking exists only on actual FTS search, not when we fetch `PgSearch::Document` in a projection